src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
 public class Application {
+
+    private static final Lottery lottery = new Lottery();
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
```

Comment:
Consider defining the number of balls as a constant field, to avoid magic numbers and increase code readability (checklist item: All magic numbers in your code should be constants).